app/src/main/res/layout-land/walkthrough_final_fragment.xml

```diff
@@ -23,7 +23,7 @@
       android:layout_width="180dp"
       android:layout_height="180dp"
       android:layout_marginStart="28dp"
-      android:src="@drawable/ic_onboarding_0"
+      android:src="@drawable/ic_landscape_onboarding_0"
```

Comment:
It is causing build failure in my system saying resource doesn't exist. Should I revert them?

Reply:
Reverted.